dask_searchcv/model_selection.py

```diff
@@ -678,6 +681,10 @@ def __init__(self, estimator, scoring=None, iid=True, refit=True, cv=None,
         self.scheduler = scheduler
         self.n_jobs = n_jobs
         self.cache_cv = cache_cv
+        if self.refit in (True, False):
+            self._score_key = 'score'
+        else:
+            self._score_key = self.refit
```

Comment:
Would prefer not to store hidden state on the class, as that's an antipattern in scikit-learn. Could implement this as a property if it cleans up the code.